<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpClass</class>
 <widget class="QWidget" name="HelpClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MARMEAS Help</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>851</width>
     <height>531</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="toc">
     <attribute name="title">
      <string>Content</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="toc_layout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="index">
     <attribute name="title">
      <string>Index</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="index_layout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="home">
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="back">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="HelpBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HelpBrowser</class>
   <extends>QTextBrowser</extends>
   <header>help_browser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
